### DISCUSSION TITLE
Can't use first_name, last_name or description in Pods form for Extended User

### DISCUSSION THREAD

**Discussion by S8nsick66** [pods-framework/pods#6625] [Q&A]
Pod "mypod", Type: User (extended), Storage Type: Meta
Pods 2.6.1
WP 4.4.1

From the docs (http://pods.io/docs/code/pods/form/):

**Test 1: field names only**

```
$mypod = pods( 'mypod' ); 
$fields = array( 'description' ); 
echo $mypod->form( $fields ); 
```

Result: Form field is not displayed, lots of "Undefined index".
Notice: Undefined index: type in /pods/ui/front/form.php on line 13
Notice: Undefined index: options in /pods/ui/front/form.php on line 13
Notice: Undefined index: label in /pods/ui/front/form.php on line 114
Notice: Undefined index: help in /pods/ui/front/form.php on line 114
Notice: Undefined index: in /pods/classes/PodsForm.php on line 212

**Test 2: array with default values**

```
$mypod = pods( 'mypod' ); 
$fields = array(
    'description' => array(
        'label' => 'Description',
        'type' => 'paragraph',
        'help' => '',
        'options' => array(
            'required' => true,
        )
    )
);
echo $mypod->form( $fields ); 
```

Result: Form is displayed with description field. On submit, alert with "Error: User data is required but is either invalid or empty"

**Test 3: array + existing pod id**

```
$mypod = pods( 'mypod', $id ); 
$fields = array(
    'description' => array(
        'label' => 'Description',
        'type' => 'paragraph',
        'help' => '',
        'options' => array(
            'required' => true,
        )
    )
);
echo $mypod->form( $fields ); 
```

Result: Form is display, with description field. On submit, redirected to "?success=1" but **the value is not saved**.

**Comment by quasel** [comment]
is this still an issue?

**Comment by jimtrue** [comment]
Related #1890
